packer/install-dependencies.sh

```diff
@@ -25,7 +25,7 @@ fi
 
 installClearContainersRuntime
 
-VNET_CNI_VERSIONS="1.0.14 1.0.15 1.0.16"
+VNET_CNI_VERSIONS="1.0.16 1.0.17"
```

Comment:
I removed the older versions, no need to keep around too many versions. Keeping 1.0.16 for easy revert so we don't have to re-publish the image if needed.